test/PowerShellEditorServices.Test.Protocol/Message/MessageReaderWriterTests.cs

```diff
@@ -20,12 +20,12 @@ public class MessageReaderWriterTests
         const string TestEventFormatString = "{{\"event\":\"testEvent\",\"body\":{{\"someString\":\"{0}\"}},\"seq\":0,\"type\":\"event\"}}";
         readonly int ExpectedMessageByteCount = Encoding.UTF8.GetByteCount(TestEventString);
 
-        private ILogger logger;
+        private IPsesLogger logger;
         private IMessageSerializer messageSerializer;
 
         public MessageReaderWriterTests()
         {
-            this.logger = new NullLogger();
+            this.logger = Logging.CreateLogger().Build();
```

Comment:
So this is the equivalent of a Null logger?

Reply:
Yeah, it's not as explicit, but Serilog with no sinks does nothing. We could make a wrapper method maybe?